tests/test_jobs_client.py

```diff
@@ -324,13 +325,13 @@ def test_list_jobs():
         assert list_jobs[0].job_id.value == "432b274a8f754968888807fe1eba237b"
         assert list_jobs[0].payload_id.value == "532b274a8f754968888807fe1eba237b"
         assert list_jobs[0].pipeline_id.value == "932b274a8f754968888807fe1eba237b"
-        assert list_jobs[0].date_created == datetime.datetime(2020, 1, 29, 12, 29, 21)
+        assert list_jobs[0].date_created == datetime.datetime(2021, 3, 8, 18, 6, 31, tzinfo=datetime.timezone.utc)

```

Comment:
What's happening here?

Reply:
I needed to update the Pytest to include the tzinfo for the assertion. I also fixed the unix epoch offset as it was slightly off (in jobs_client). That previous test timestamp was invalid.